src/daemon/rpmostreed-transaction-types.cxx

```diff
@@ -1069,12 +1104,12 @@ deploy_transaction_execute (RpmostreedTransaction *transaction,
       glnx_close_fd (&self->local_repo_remote_dfd);
     }
 
-  g_autofree gchar *new_refspec = NULL;
-  g_autofree gchar *old_refspec = NULL;
+  g_autofree gchar *new_refspecdata = NULL;
+  g_autofree gchar *old_refspecdata = NULL;
```

Comment:
What's going on with this hunk?  Is it a clarification of the variable name?  Or now we need to track more data?  Particularly if the former, can any "prep work" be factored out here into a separate PR?

Reply:
It is the former. Yep, I'll refactor things like this in a separate commit. (I also think it'd be nice to rename some other variables around here as well; I recall there were various places where what `refspec` referred to was ambiguous)